src/integration/pages/articles/imageCopyrightNonBBC.js

```diff
@@ -0,0 +1,18 @@
+export default () => {
+  describe('Image Copyright Non BBC', () => {
+    const copyrightTextEl = document.querySelector(
+      'main figure span:nth-child(2)',
+    );
+    it('should be in the document', () => {
+      expect(copyrightTextEl).toBeInTheDocument();
+    });
+
+    it('should have the right text content', () => {
+      expect(copyrightTextEl).toHaveTextContent('Getty Images');
+    });
```

Comment:
I wonder if we should remove this, as the content is being captured in the next text. It also means that the test could be reused for other non-BBC copyright holders, not just Getty Images

Reply:
True, I can just check for existence and have the snapshot cover the content